src/coprocessor/statistics/analyze.rs

```diff
@@ -31,11 +30,13 @@ use super::cmsketch::CmSketch;
 use super::fmsketch::FmSketch;
 use super::histogram::Histogram;
 use crate::coprocessor::dag::TiKVStorage;
+use crate::coprocessor::MEMTRACE_ANALYZE;
 use crate::coprocessor::*;
 use crate::storage::{Snapshot, SnapshotStore, Statistics};
 
 const ANALYZE_VERSION_V1: i32 = 1;
 const ANALYZE_VERSION_V2: i32 = 2;
+const ANALYZE_MAX_TIME_SLICE: Duration = Duration::from_secs(1);
```

Comment:
@sticnarf here I think analyze task can run long time, so I change the time slice from `1ms` to `1s`. Please take a look whether it's ok/necessary or not.

Reply:
Time slice should not be that long. Otherwise it may occupy the thread for too long and short tasks will be affected.
Short time slice does not mean the task is only allowed to run that short